drivers/wifi/nrf700x/zephyr/src/qspi/src/rpu_hw_if.c

```diff
@@ -311,11 +302,13 @@ int rpu_enable(void)
 	rpu_gpio_config();
 	rpu_pwron();
 	rpu_qspi_init();
+	/* WAR for EK boards to wait longer for RPU to be ready after
+	 * powering it on.
+	 */
+	k_msleep(8);
```

Comment:
Should this be `SLEEP_TIME_MS` from `rpu_hw_if.h`?

Reply:
Yes, in fact its a slightly different WAR that I have raised in https://github.com/nrfconnect/sdk-nrf/pull/8453 will use that.